APMrover2/Log.cpp

```diff
@@ -40,18 +40,28 @@ void Rover::Log_Write_Performance()
 struct PACKED log_Steering {
     LOG_PACKET_HEADER;
     uint64_t time_us;
-    float demanded_accel;
-    float achieved_accel;
+    int16_t steering_in;
+    float steering_out;
+    float desired_lat_accel;
+    float lat_accel;
+    float desired_turn_rate;
+    float turn_rate;
 };
 
 // Write a steering packet
 void Rover::Log_Write_Steering()
 {
+    float lat_accel = 0.0f;
```

Comment:
Set this to `dataflash.quiet_nanf()` since `0.0f` is in-band

Reply:
thanks.  fixed in follow up commit.